src/DashboardHub/Bundle/AppBundle/Resources/config/routing.yml

```diff
@@ -2,6 +2,10 @@ dashboardhub_app_homepage:
     path:     /
     defaults: { _controller: DashboardHubAppBundle:Default:index }
 
+dashboardhub_app_dashboard:
+    path:     /d/{uid}
```

Comment:
you can technically use dashboard/ for all if you are cunning about your
routing

On Fri, Mar 13, 2015 at 7:26 AM, Eddie Abou-Jaoude <notifications@github.com
> wrote:

> In src/DashboardHub/Bundle/AppBundle/Resources/config/routing.yml
> <https://github.com/DashboardHub/PipelineDashboard/pull/75#discussion_r26368596>
> :
>
> > @@ -2,6 +2,10 @@ dashboardhub_app_homepage:
> >      path:     /
> >      defaults: { _controller: DashboardHubAppBundle:Default:index }
> >
> > +dashboardhub_app_dashboard:
> > +    path:     /d/{uid}
>
> This would become:
>
> dashboardhub.io/dashboard/{key}
> <http://dashboardhub.io/dashboard/%7Bkey%7D>
> and
> dashboardhub.io/dashboards/
> dashboardhub.io/dashboards/add
> dashboardhub.io/dashboards/edit
>
> You don't think it is too long & the word dashboard repeated too many
> times?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DashboardHub/PipelineDashboard/pull/75/files#r26368596>
> .
>